extensions/positron-python/src/client/pythonEnvironments/creation/provider/uvUtils.ts

```diff
@@ -7,7 +7,7 @@ import { CancellationToken, QuickPickItem } from 'vscode';
 import { showQuickPickWithBack } from '../../../common/vscodeApis/windowApis';
 import { CreateEnv } from '../../../common/utils/localize';
 
-const SUPPORTED_UV_PYTHON_VERSIONS = ['3.13', '3.12', '3.11', '3.10', '3.9'];
+const SUPPORTED_UV_PYTHON_VERSIONS = ['3.13', '3.12', '3.11', '3.10', '3.9', '3.14'];
```

Comment:
I put this at the bottom and added (preview) for now; see discussion in https://github.com/posit-dev/positron/issues/9654

<img width="1422" height="1214" alt="Image" src="https://github.com/user-attachments/assets/0294cd94-0768-432d-b09f-200dbc152ea1" />